client/src/components/App.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import axios from "axios";
+import Header from "./Overview/Header/Header.jsx";
+import SearchInput from "./Overview/Search/SearchInput.jsx";
+import OverviewResults from "./Overview/OverviewResults.jsx";
+import FilterSearchResult from "./Overview/Search/FilterSearchResult.jsx";
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      filteredResult: [],
+      volunteerList: [],
+      selectedVolunteer: {}
+    };
+
+    this.handleSearchChange = this.handleSearchChange.bind(this);
+    this.selectOneVolunteer = this.selectOneVolunteer.bind(this);
+    this.clearSearchInput = this.clearSearchInput.bind(this);
+  }
+
+  componentDidMount() {
+    this.getAllVolunteers();
+  }
+
+  // query all the volunteers, called when loading the page
+  getAllVolunteers() {
+    axios.get(`http://localhost:3030/api/person`)
+      .then(res => {
+        this.setState({
+          filteredResult: res.data.data,
+          volunteerList: res.data.data
+        });
+      })
+      .catch((error) => {
+        console.log(error);
+      })
+  }
+
+  // query volunteer information for each individual volunteer with their id
+  selectOneVolunteer(id) {
+    axios.get(`http://localhost:3030/api/person/${id}`)
+      .then(res => {
+        this.setState({
+          selectedVolunteer: res.data.data
+        }, () => {
+          const { selectedVolunteer } = this.state;
+          console.log(`Name: ${selectedVolunteer["first_name"]} ${selectedVolunteer["last_name"]}\n\
```

Comment:
Can we take this console.log out? 
Or is this showing that the click is working and will be removed when the report card piece is finished?

Reply:
Yes, this is just showing the click is working